spec/controllers/account_controller_spec.rb

```diff
@@ -69,7 +68,7 @@
       end
 
       it 'should not redirect to logout' do
-        post :login , :username => admin.login, :password => 'adminADMIN!', :back_url => '/logout'
+        post :login, username: admin.login, password: 'adminADMIN!', back_url: '/logout'
```

Comment:
Line is too long. [88/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.